modules/ocf_printhost/files/enforcer

```diff
@@ -164,6 +173,16 @@ def send_printer_mail(message, job, quo):
 
 def prehook(c, r, job):
     quo = quota.get_quota(c, job.user)
+
+    # A stop gap until we figure out why some file types aren't printing correctly
+    for file_source, problematic in PROBLEMATIC_FILE_SOURCE:
+        if problematic(job):
+            msg = NOTIFY_PROBLEMATIC_FILE_SOURCE.format(
+                file_source=file_source,
+            )
+            r.publish(job.user, msg)
+            sys.exit(255)
```

Comment:
Nit: some exit codes have special meanings. You may wish to consider picking one which would not clash with any of these.

Context: https://www.tldp.org/LDP/abs/html/exitcodes.html#EXITCODESREF